src/helpers/api.js

```diff
@@ -0,0 +1,5 @@
+export const buildBackendURL = route => {
+  const protocol = (window.location.protocol === "https:" ? "https://" : "http://");
+  const baseUrl = (window.location.hostname === "localhost" ? "localhost:8000" : "co2-calculator-api.herokuapp.com");
+  return protocol + baseUrl + route;
```

Comment:
We can use a [protocol-relative URL](https://www.paulirish.com/2010/the-protocol-relative-url/) here starting with two forward slashes. For example `//co2-calculator-api.herokuapp.com`.